wazo_confd/plugins/outcall/resource.py

```diff
@@ -14,13 +15,30 @@
 class OutcallList(ListResource):
     model = Outcall
     schema = OutcallSchema
+    outcall_name_fmt = 'outcall-{tenant_slug}-{outcall_uuid}'
+
+    def __init__(self, tenant_dao, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._tenant_dao = tenant_dao
 
     def build_headers(self, outcall):
         return {'Location': url_for('outcalls', id=outcall.id, _external=True)}
 
     @required_acl('confd.outcalls.create')
     def post(self):
-        return super().post()
+        form = self.schema().load(request.get_json())
+        form = self.add_tenant_to_form(form)
+
+        tenant = self._tenant_dao.get(form['tenant_uuid'])
+        # NOTE(afournier): we use a UUID as if it was the outcall UUID but it's not
+        # Outcalls do not use UUIDs yet
+        form['name'] = self.outcall_name_fmt.format(
+            tenant_slug=tenant.slug,
+            outcall_uuid=uuid4(),
```

Comment:
same comment as: https://github.com/wazo-platform/wazo-confd/pull/401#discussion_r1266700926
About adding uuid in database